.travis.sh

```diff
@@ -44,11 +35,12 @@ function build ()
   prepare
 
   cmake $SOURCE_DIR \
-      -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE \
+      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Debug} \
```

Comment:
My CMake is still a work in progress but I'm guessing this is akin to a default argument if `CMAKE_BUILD_TYPE` is not defined?

Reply:
This is bash variable substitution